pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -572,8 +573,11 @@ func convertToPrincipal(subject *rbacproto.Subject) *policyproto.Principal {
 		}
 	}
 
-	if subject.Group != "" {
-		rbacLog.Errorf("ignored Subject.group %s, not implemented", subject.Group)
+	if subject.Group != "" && subject.Properties[attrRequestClaimGroups] == "" {
```

Comment:
This condition says, if there is a "request.auth.claims[groups]" property, "subject.group" is not honored. I think we should do it the opposite way, "subject.group" is always honored. If "subject.group" exist, users should not define another condition with "request.auth.claims[groups]". Note that this is the way we handle "subject.user".

Reply:
Done.